### ISSUE TITLE
[userTypeFix] Code Coverage: Create tests for src/screens/Users/Users.tsx

### ISSUE THREAD

**Issue by varshith257** [PalisadoesFoundation/talawa-admin#1770]
We will need unittests done for all methods, classes and functions found in this file.

Any widgets, components, modals referenced in this file must also have unittests done

PR Acceptance Criteria

1. When complete this file must show at least 100% coverage when merged into the code base.
2. The PR will show a report for the code coverage for the file you have added. You can use that as a guide

**Comment by varshith257** [comment]
@Shubh152 Get it assigned and submit PR once it is done

**Comment by Shubh152** [comment]
please assign this to me

**Comment by dhanagopu** [comment]
@Shubh152  Our policy is to assign no more than two issues to each contributor across all repositories. You have reached your limit, please wait until your existing issues are closed before requesting more issues.

**Comment by varshith257** [comment]
@Shubh152 Any update on this? Because of this many PRs failing test.

**Comment by Shubh152** [comment]
Working on it. Will try to fix it by the end of the day.

**Comment by palisadoes** [comment]
We are tracking this issue with one of our projects, and we'd like to track its progress especially as the GSoC announcement dates draw near.

1. When you create your PR, please link it back to this project:
    1. https://github.com/orgs/PalisadoesFoundation/projects/24/views/1
1. The process is simple. It can be found here:
    1. https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#manually-linking-a-pull-request-to-an-issue-using-the-pull-request-sidebar

**Comment by varshith257** [comment]
@Shubh152 Any update on this?

**Comment by Cioppolo14** [comment]
unassigning due to no activity or PR
